BUILD.bazel

```diff
@@ -4,6 +4,8 @@ load("//bazel:common_settings.bzl", "string_flag")
 # OS-CPU CONFIG SETTINGS
 #
 
+load("@hedron_compile_commands//:refresh_compile_commands.bzl", "refresh_compile_commands")
+
```

Comment:
All `load()` statements in `BUILD` files should be grouped together at the top of the file. See https://bazel.build/build/style-guide#file-structure.